app/controllers/users_controller.rb

```diff
@@ -13,4 +13,9 @@ def create
       render 'new'
     end
   end
+
+  def my_queue
```

Comment:
There is no need to have empty methods in the controller. As long as there is a template by the same name, you should not have to create an empty method.